src/parser/test/ParserTest.cpp

```diff
@@ -307,6 +307,77 @@ TEST(Parser, EdgeOperation) {
         ASSERT_TRUE(result.ok()) << result.status();
     }
 }
+// Column space format test, expected SyntaxError
+TEST(Parser, ColumnSpacesTest) {
+    {
+        GQLParser parser;
+        std::string query = "CREATE TAG person(name, age, married bool)";
+        auto result = parser.parse(query);
+        ASSERT_FALSE(result.ok()) << result.status();
```

Comment:
Be careful about `StatusOr`. It holds either a _value_ or _Status_, but never both.

The same to all other places.